pkg/segment/reader/segread/segstatsreader.go

```
@@ -303,7 +302,7 @@ func getRange(max utils.CValueEnclosure, min utils.CValueEnclosure) *utils.CValu
 			return &utils.CValueEnclosure{}
 		}
 	default:
-		log.Errorf("getRange: unsupported dtype: %v", max.Dtype)
+		log.Debugf("getRange: unsupported dtype: %v", max.Dtype)
 	}
```

Silently failing here will likely make this harder to debug. We should probably return some error (or a nil CValueEnclosure) or keep the error log